modules/pull/merge.go

```diff
@@ -310,21 +310,6 @@ func Merge(pr *models.PullRequest, doer *models.User, baseGitRepo *git.Repositor
 		l.PushFront(mergeCommit)
```

Comment:
so it's unnecessary from line 295

Reply:
Yes, I need to also test every merge style to be sure that it trigger the push via post-receive